src/services/user.service.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const Users = require('../db/models/users');
+
+function normalizeUser({ id, name }) {
```

Comment:
Consider using a more descriptive name for the input parameter, like `user` instead of `{ id, name }`.